docs/data/charts/tooltip/tooltip.md

```diff
@@ -97,6 +97,21 @@ It removes the header showing the x-axis value from the tooltip.
 To hide a series, the formatted value should be `null`.
 To display the series with a blank space, return an empty string.
 
+## Position
+
+By default the tooltip is placed relative to the pointer position.
+If the pointer is not available, it is placed relative to the item instead.
+
+This behavior can be modified with the `anchor`, `position`, and `placement` props.
+
+- The `anchor: 'pointer' | 'node'` indicates if the tooltip should be placed relative to the pointer or the node.
+- The `position: 'top' | 'right' | 'bottom' | 'left'` precise the anchor position compared to the node.
+- The `placement` is the tooltip placement from [PopperJS](https://popper.js.org/docs/v1/#popperplacements--codeenumcode). By default the same value as `position` if defined.
```

Comment:
Tried to make this clearer. Also updated the link to Popper v2 which is the version we're using.

```suggestion
- The `position: 'top' | 'right' | 'bottom' | 'left'` defines the anchor position compared to the node. This prop has no effect if the anchor is the pointer.
- The `placement` is the tooltip placement from [PopperJS](https://popper.js.org/docs/v2/constructors/#options). It specifies the tooltip position in relation to the anchor position. By default the same value as `position` if defined.


For example, setting `anchor: 'node'`, `position: 'bottom'` and `placement: 'top'` on a bar chart would render a tooltip above the bottom of a bar.
```